engine/value.c

```diff
@@ -224,7 +224,7 @@ char *value_get_string(engine_value *val) {
 		break;
 	default:
 		value_copy(&tmp, val->internal);
-		convert_to_cstring(&tmp);
```

Comment:
Same as above, this is only available from PHP version 5.5 and up. However, `convert_to_string` has different semantics with regards to floating-point numbers, which may cause issues if left unchecked.